### PR TITLE
Add event to allow modification of dompdf options before render

### PR DESCRIPTION
**Pull request by pdaleramirez**
(#1761)

### CODE REVIEW

File: src/services/Pdfs.php [hunk 1]
```diff
@@ -146,6 +147,29 @@ class Pdfs extends Component
...
+     * @since 3.x
+     *
+     * ```php
+     * use craft\commerce\events\PdfRenderOptionsEvent;
+     * use craft\commerce\services\Pdfs;
+     * use yii\base\Event;
+     *
+     * Event::on(
+     *     Pdfs::class,
+     *    Pdfs::EVENT_ADD_RENDER_OPTIONS,
```
Comment (lukeholder) [inline src/services/Pdfs.php:161]: Rename to `EVENT_MODIFY_RENDER_OPTIONS`

**Review by lukeholder — CHANGES_REQUESTED** [review]

File: src/services/Pdfs.php (hunk 1 above)
Comment (pdaleramirez) [reply src/services/Pdfs.php]: Done renaming the event name

**Review by nfourtythree — APPROVED** [review]
This all looks good to me, as long as the event is not cancellable which it doesn't look like it is 👍

**Comment by lukeholder** [comment]
Thanks!

**Pull request merged**